.github/workflows/service_test_azblob.yml

```diff
@@ -75,7 +75,7 @@ jobs:
 
   azure_azblob:
     runs-on: ubuntu-latest
-    if: github.event_name == 'push' || !github.event.pull_request.head.repo.fork
+    if: (github.event_name == 'push' && github.repository == 'apache/incubator-opendal') || !github.event.pull_request.head.repo.fork
```

Comment:
Seems failed for 
![image](https://github.com/apache/incubator-opendal/assets/5351546/44bb382c-e554-44b6-9d73-9d890a9d6695)

Reply:
Thanks. Let me check what happend.

Reply:
It not this branch's problem. I solved it at the branch `dqhl76:add-restrictions-ci`.(pull request branch) 
The problem happened at the branch `dqhl76:add-restriction-ci`. 
I am trying GitHub codespace, so I made a mistake when I uploaded my code.